frontend/src/components/common/UserManagement/ConfirmCancelEditModal.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  Button,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from "@chakra-ui/react";
+import React from "react";
+
+import useViewport from "../../../hooks/useViewport";
+
+interface ConfirmCancelEditModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  discardChanges: () => void;
+}
+const ConfirmCancelEditModal = ({
+  isOpen,
+  onClose,
+  discardChanges,
+}: ConfirmCancelEditModalProps) => {
+  const { isDesktop } = useViewport();
```

Comment:
We don't need to use the hook in this case since we're not conditionally passing values it's all styling
```suggestion
```